server/plugins/Convolution.cpp

```diff
@@ -766,6 +775,10 @@ void StereoConvolution2L_Ctor(StereoConvolution2L *unit)
 		unit->m_prevtrig = 0.f;
 
 		SETCALC(StereoConvolution2L_next);
+
+		// initialize outputs
+		OUT0(0) = IN0(0);
+		OUT0(1) = IN0(1);
```

Comment:
I don't understand this line, but _warning_ I have no big clue about how the UGen code works. I'm just thinking that `OUT0(1)` might be the second sample of the first (left) output and not (as intended?) the first sample of the second (right) output. Also, the input to `StereoConvolution` is monophonic, so `IN0(1)` would probably also not be correct?